src/app/components/AssignmentForm.tsx

```diff
@@ -56,26 +71,55 @@ export interface AssignmentFormProps {
+                {showCourtSecurityFields && <h1> Courts</h1>}
+                {showDocumentSericesFields && <h1> Document Services </h1>}
+                {showEscortServicesFields && <h1> Escorts</h1>}
+                {showGateSecurityFields && <h1> Gates</h1>}
+                {showOtherAssignmentFields && <h1> Other Assignment </h1>}
                 <Form onSubmit={handleSubmit}>
-                    { showCourtSecurityFields && <CourtSecurityFields /> }
-                    { showEscortServicesFields && <EscortServiceFields />}
-                    { showGateSecurityFields && <GateSecurityFields /> }
-                    <Field name="startTime" component={DateTimeField} label="Start Time" validate={[Validators.required]}/>
-                    <Field name="endTime" component={DateTimeField} label="End Time" validate={[Validators.required]}/>
-                    <Field name="abilities" component={RequiredTrainingChecklist} label="Required Qualifications" />
-                    <Field name="sherrifsRequired" component={NumberOfSheriffsSelector} label="Number of Sheriffs Required"/>
-                    <Field name="notes" component={TextArea} label="Notes" />
+                    {showCourtSecurityFields && <CourtSecurityFields />}
+                    {showEscortServicesFields && <EscortServiceFields />}
+                    {showGateSecurityFields && <GateSecurityFields />}
+                    <Field name="sherrifsRequired" component={TextField} label="Number of Sheriffs Required" validate={[Validators.required, Validators.integer]} />
+                    {!isDefaultTemplate &&
+                        <div>
+                            <Field name="startTime" component={DateTimeField} label="Start Time" validate={[Validators.required]} />
+                            <Field name="endTime" component={DateTimeField} label="End Time" validate={[Validators.required]} />
+                            <Field name="abilities" component={RequiredTrainingChecklist} label="Required Qualifications" />
+                            <Field name="notes" component={TextArea} label="Notes" />
+                        </div>
+                    }
+                    {isDefaultTemplate &&
+                        <div>
+                            <strong>Days &amp; Times</strong>
+                            <RecurrenceFieldArray name="recurrenceInfo" component={(p) => {
+                                const { fields } = p;
+                                return (
+                                    <ListGroup >
+                                        {fields.map((recurrenceInfoFieldName, index) => {
+                                            return (
+                                                <ListGroupItem key={index}>
+                                                    <Button bsStyle="danger" onClick={() => fields.remove(index)} className="pull-right"><Glyphicon glyph="trash" /></Button><br />
```

Comment:
Already chatted about this as well, I think containing some of this in a separate render method might be good.